### ISSUE TITLE
Rough start. No benefits on Azure? :)

### ISSUE THREAD

**Issue by jgwinner** [Azure/iisnode#101]
I stumbled on this repository, and I clicked on "Who uses it?"

The first line goes to:

http://blogs.msdn.com/b/hanuk/archive/2012/05/05/top-benefits-of-running-node-js-on-windows-azure.aspx 

Which is is a dead link :(

So then from the Readme, I go to the wiki, which mentions Getting Started; if I click on that, I go to the old site! Ah - turns out the entire Readme refers to the old site.

I'm a programmer, so normally I figure the docs are in the .js file or the .h file, but in this case, running into two dead or depreciated links doesn't make the product look really strong - and at a time that Node.JS is running EVERYWHERE.

Basically - the Readme needs a going over. 

        == John ==

**Comment by iqfy-kyle** [comment]
https://github.com/Azure/iisnode/issues/76

**Comment by lextm** [comment]
MSDN blogs have been archived, so the right link to read that old post is https://learn.microsoft.com/en-us/archive/blogs/hanuk/top-benefits-of-running-node-js-on-windows-azure

HttpPlatformHandler has been a key alternative to host Node.js web apps on IIS/Azure, https://docs.lextudio.com/blog/the-rough-history-of-iis-httpplatformhandler/ So IISNode isn't your only option.
